src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    limit to allocate tuples, but there is additional memory used to store
+    indexes and connection information.
+]])
+
+M['memtx.min_tuple_size'] = format_text([[
+    Size of the smallest allocation unit in bytes. It can be decreased if
+    most of the tuples are very small.
+]])
+
+M['memtx.slab_alloc_factor'] = format_text([[
+    The multiplier for computing the sizes of memory chunks that tuples
+    are stored in. A lower value may result in less wasted memory depending
+    on the total amount of memory available and the distribution of item sizes.
+]])
+
+M['memtx.slab_alloc_granularity'] = format_text([[
+    Specify the granularity in bytes of memory allocation in the small
+    allocator. The `memtx.slab_alloc_granularity` value should meet the
+    following conditions:
+
+    - The value is a power of two.
+    - The value is greater than or equal to 4.
+
+    Below are few recommendations on how to adjust the
+    `memtx.slab_alloc_granularity option`:
+
+    - If the tuples in space are small and have about the same size, set the
+    option to 4 bytes to save memory.
+    - If the tuples are different-sized, increase the option value to allocate
+    tuples from the same `mempool` (memory pool).
+]])
+
+M['memtx.sort_threads'] = format_text([[
+    The number of threads from the thread pool used to sort keys of secondary
+    indexes on loading a `memtx` database. The minimum value is 1, the maximum
+    value is 256. The default is to use all available cores.
+]])
+
+-- }}} memtx configuration
+
+-- {{{ metrics configuration
+
+M['metrics'] = format_text([[
+    The `metrics` section defines configuration parameters for metrics.
+]])
+
+M['metrics.exclude'] = format_text([[
+    An array containing the metrics to turn off. The array can contain the same
+    values as the `exclude` configuration parameter passed to `metrics.cfg()`.
+]])
+
+M['metrics.exclude.*'] = 'Metrica name.'
+
+M['metrics.include'] = format_text([[
+    An array containing the metrics to turn on. The array can contain the same
+    values as the `include` configuration parameter passed to `metrics.cfg()`.
+]])
+
+M['metrics.include.*'] = 'Metrica name.'
+
+M['metrics.labels'] = 'Global labels to be added to every observation.'
+
+M['metrics.labels.*'] = 'Label name.'
+
+-- }}} metrics configuration
+
+-- {{{ process configuration
+
+M['process'] = format_text([[
+    The `process` section defines configuration parameters of the Tarantool
+    process in the system.
+]])
+
+M['process.background'] = format_text([[
+    Run the server as a daemon process.
+
+    If this option is set to true, Tarantool log location defined by the
+    `log.to` option should be set to file, pipe, or syslog - anything other
+    than stderr, the default, because a daemon process is detached from a
+    terminal and it can't write to the terminal's stderr. Warn: Do not enable
+    the background mode for applications intended to run by the tt utility.
```

Comment:
I would move the warning to its own paragraph.

Reply:
Done.
```lua
I['process.background'] = format_text([[
    Run the server as a daemon process.

    If this option is set to true, Tarantool log location defined by the
    `log.to` option should be set to file, pipe, or syslog - anything other
    than stderr, the default, because a daemon process is detached from a
    terminal and it can't write to the terminal's stderr.

    Warn: Do not enable the background mode for applications intended
    to run by the tt utility.
]])
```